test/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from src.play_scraper import PlayPageScraper
+
+
+def pytest_configure():
+    pytest.FILE = "file:"
+
+
+@pytest.fixture(scope="session")
+def icon_dir(tmp_path_factory):
+    return tmp_path_factory.mktemp("icon_dir")
+
+
+@pytest.fixture(scope="session")
```

Comment:
Your tests should be independent of order and you should not reuse the same test directory for multiple tests, so I would remove the `scope` here and in `icon_dir`